libparsec/crates/protocol/schema/authenticated_cmds/invite_list.json5

```diff
@@ -61,6 +61,28 @@
                                 "type": "InvitationStatus"
                             }
                         ]
+                    },
+                    {
+                        "name": "ShamirRecovery",
```

Comment:
I'm expecting to see a `introduced_in` field, it's something that was forgotten or it's something else ?

Reply:
https://github.com/Scille/parsec-cloud/issues/8878
> The purpose and usage of the introduced_in and introduced_in_revision fields should be explained in a future RFC once API 5 is out.